birdhouse/env.local.example

```diff
@@ -142,6 +142,8 @@ export POSTGRES_MAGPIE_PASSWORD=postgres-qwerty
 # "Notebook" are all the tutorial notebooks on Jupyter.
 #export AUTODEPLOY_NOTEBOOK_FREQUENCY="@every 5m"
 
+#export NOTEBOOK_DEST_DIR=/tmp/notebooks
```

Comment:
I don't think this NOTEBOOK_DEST_DIR would ever need to be manually set in this `env.local` at all.

It is always set in the "loop" no?  If you need default value for the loop, at least keep it really close to the loop, not alone like here.

Reply:
Moved to the env file in the pavics-jupyter-base repo